src/cmd/sign.js

```diff
@@ -117,6 +155,8 @@ export function getIdFromSourceDir(sourceDir: string): Promise<string> {
 }
 
 
+// getIdFromSourceDir helper implementation.
```

Comment:
This comment doesn't seem right. I don't think these marker comments are necessary when there are no proceeding type definitions.